src/adapter/etl-adapter-google-sheet/src/Flow/ETL/DSL/GoogleSheet.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\DSL;
+
+use Flow\ETL\Adapter\GoogleSheet\GoogleSheetExtractor;
+use Flow\ETL\Adapter\GoogleSheet\GoogleSheetRange;
+use Flow\ETL\Extractor;
+use Google\Service\Sheets;
+
+class GoogleSheet
+{
+    /**
+     * @param string $spreadsheetIds
+     * @param int $rows_in_batch
+     * @param bool $with_header
+     *
+     * @return Extractor
+     */
+    final public static function from(
+        string $spreadsheetIds,
+        Sheets $service,
```

Comment:
Sheets are google main class used to communicate with google sheets, I would probably instantiate it inside of Loader/Extractor than here based on input parameters. Look at doctrine DBAL how it's done there

Reply:
ok